src/gui/mzroll/mainwindow.cpp

```diff
@@ -2253,7 +2253,7 @@ void MainWindow::readSettings() {
 
     //Pull Isotopes in options
     if (!settings->contains("isotopeC13Correction"))
-        settings->setValue("isotopeC13Correction", 2);
+        settings->setValue("isotopeC13Correction", 0);
```

Comment:
this can still be a problem on the user's system, no ? if they don't uninstall or remove 'Application Settings.conf'  that value won't change for them

Reply:
@rish9511 Will this be a problem with all default settings that are not part of mavenParameters?

Reply:
@rish9511 Added variable to mavenParameters.